src/nl/Stratego/Bord.java

```diff
@@ -75,20 +75,30 @@ public void move(int x, int y, int x_new, int y_new){
         }
     }
 
-    public String toString(){
+    //hieronder wordt het hele bord geprint, zie volgende methode voor team specifiek bord printen
+
+    public void bordPrinten(){
         StringBuilder bordstring = new StringBuilder();
-        bordstring.append("+---+---+---+---+---+---+---+---+---+---+\n");
+        bordstring.append("X:  1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10| \n"); //deze coordinaten worden geprint boven het bord
+        bordstring.append("  -----------------------------------------\n"); // dit is een afscheiding van coordinaten tov gevulde matrix
+        bordstring.append("Y:+---+---+---+---+---+---+---+---+---+---+\n");
             for (int y = 0; y < 10; y++) {
-                for (int x = 0; x < 10; x++) { //deze forloop voegt voor ieder vakje de value van het spelstuk toe of een "o" als het vakje leeg is.
+                int yCoordinaat = y+1;//deze y-coordinaat wordt gedefinieerd zodat deze geprint kan worden als coordinatenstelsel
+                if(yCoordinaat<10) {
```

Comment:
In de fucntie met bordPrinten(int huidig team) wordt dit stukje code in de eerste for loop behandeld. Als ik je branch download dan staat deze ook alleen in de eerste for loop. Even controleren hoe dat zit voordat we hem mergen

Reply:
gisteren opgelost - misverstand